packages/react-native-bridge/package.json

```diff
@@ -12,9 +12,8 @@
     "react-native"
   ],
   "author": "",
-  "license": "",
+  "license": "GPL-2.0-or-later",
   "peerDependencies": {
-    "react-native": "0.59.0",
-    "react-native-windows": "0.54.0"
+    "react-native": "*"
```

Comment:
Should `react` be listed here as well? I see it listed in another package. Given that `RN` depends on React, we probably can remove it on another package, what do you think?

Reply:
I'm not sure to be honest, we haven't used or thought about `react-native-bridge` as a standalone package. It's still tightly coupled with the main native editor, thus the reason why I replaced the version with `"*"` here, so we can revisit later.
We could indeed add `react` here, but I wonder if it would change anything in termes of packages that use this bridge dependency, since `react` is already a peer dependency of `react-native` it should display an error or a warning if react is not a dependency.

Reply:
Indeed `react` is a dependency of `react-native-aztec`, so it could probably be removed there. Let's revisit peerDependencies later if that's ok with you?

Reply:
Sure thing, I just flagged items to go through during the final iteration 👍